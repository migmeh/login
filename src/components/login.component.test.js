import React, { Component } from "react";
import { shallow } from 'enzyme';
import Login from "./login.component";


describe('login component',()=>{

    it('login', ()=>{
        const wrapper = <Login/>;
        expect(wrapper).toMatchSnapshot();
    });


    it('shallow login', ()=>{
        const wrapper = shallow(<Login/>);
        expect(wrapper).toMatchSnapshot();
    });



});

