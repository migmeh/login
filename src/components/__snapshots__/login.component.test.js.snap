// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login component login 1`] = `<Login />`;

exports[`login component shallow login 1`] = `
<div
  className="col-md-12"
>
  <div
    className="card card-container"
  >
    <Form(t)
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label
          htmlFor="username"
        >
          Username: 
          <strong>
            root
          </strong>
        </label>
        <Control(s)
          className="form-control"
          name="username"
          onChange={[Function]}
          type="text"
          validations={
            Array [
              [Function],
            ]
          }
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="password"
        >
          Password: 
          <strong>
            123456
          </strong>
        </label>
        <Control(s)
          className="form-control"
          name="password"
          onChange={[Function]}
          type="password"
          validations={
            Array [
              [Function],
            ]
          }
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <button
          className="btn btn-primary btn-block"
          disabled={false}
        >
          <span>
            Login
          </span>
        </button>
      </div>
      <Button(s)
        style={
          Object {
            "display": "none",
          }
        }
      />
    </Form(t)>
  </div>
</div>
`;
